maro/data_lib/oncall_routing/oncall_order_generator.py

```diff
@@ -37,13 +37,14 @@ def __init__(self, csv_path: str) -> None:
         df = pd.read_csv(csv_path, sep=',')
         buff = []
         for e in df.to_dict(orient='records'):
-            order = Order()
-            order.id = str(next(GLOBAL_ORDER_COUNTER))
-            order.coord = (e["LAT"], e["LNG"])
-            order.open_time = e["READYTIME"]
-            order.close_time = e["CLOSETIME"]
+            order = Order(
+                order_id=str(next(GLOBAL_ORDER_COUNTER)),
+                coordinate=Coordinate(e["LAT"], e["LNG"]),
+                open_time=e["READYTIME"],
```

Comment:
- convert to int time index;
- assertion in the initialization function of order